packages/rum-core/src/rawRumEvent.types.ts

```diff
@@ -204,6 +204,12 @@ export interface ViewContext extends Context {
   }
 }
 
+export interface ViewUrlContext extends Context {
+  view: {
+    url: string
```

Comment:
I found it weird to still have it in `ViewContext` then, could we handle it in a single spot?